server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/MessageSenderTest.java

```diff
@@ -0,0 +1,107 @@
+package org.apache.james.jmap.methods;
```

Comment:
licence

Reply:
I'll add it once the build is done if there's no other comment that require a fixup.